scripts/issue/1139/acl_reorg.py

```diff
@@ -0,0 +1,198 @@
+        ROOT_VALIDATOR_ROLE: r.badger_wallets["ops_root-validator_v3"],
+    },
+    "rewardsLogger": {
+        DEFAULT_ADMIN_ROLE: r.badger_wallets.dev_multisig,
+        MANAGER_ROLE: r.badger_wallets.techops_multisig,
+    },
+    "guardian": {
+        DEFAULT_ADMIN_ROLE: r.badger_wallets.dev_multisig,
+        APPROVED_ACCOUNT_ROLE: DEVELOPER_ROLE_MEMBERS,
+    },
+    "keeperAccessControl": {
+        DEFAULT_ADMIN_ROLE: r.badger_wallets.dev_multisig,
+        EARNER_ROLE: techops_signers,
+        HARVESTER_ROLE: techops_signers,
+        TENDER_ROLE: techops_signers,
+    },
+}
+
+TARGET_CONFIG_TECHOPS = {
+    "registryAccessControl": {
+        DEFAULT_ADMIN_ROLE: r.badger_wallets.techops_multisig,
+        DEVELOPER_ROLE: techops_signers,
+    },
+}
+
+
+def sim():
+    chain_id = chain.id
+    if chain_id == 1:
+        # 1. ops_multisig_old -> grant role `ADMIN_ROLE` in rewardsLogger to dev_multisig
+        ops_multisig_granting(sim=True)
+    elif chain_id == 42161:
+        # 1. dev_multisig_deprecated -> grant role `ADMIN_ROLE` in guardian to dev_multisig
+        deprecated_dev_granting(sim=True)
+    # 2. sim atomic tx
+    main("dev.badgerdao.eth", sim=True)
+
+
+def ops_multisig_granting(sim=False):
+    ops_multisig_old = GreatApeSafe(r.badger_wallets.ops_multisig_old)
+
+    reward_logger = ops_multisig_old.contract(r.rewardsLogger)
+    reward_logger.grantRole(DEFAULT_ADMIN_ROLE, dev_msig)
+
+    if not sim:
+        ops_multisig_old.post_safe_tx(call_trace=True)
+
+
+# NOTE: replace your address here with rights to post tx into dev_multisig_deprecated
+ACCOUNT_NAME = ""
+
+
+def deprecated_dev_granting(sim=False):
+    if sim:
+        dev = accounts.at(r.badger_wallets.ops_deployer4, force=True)
+    else:
+        dev = accounts.load(ACCOUNT_NAME)
+
+    safe = interface.IMultisigWalletWithDailyLimit(
+        r.badger_wallets.dev_multisig_deprecated
+    )
+
+    registry = Contract(r.registry_v2)
+    guardian = Contract(registry.get("guardian"))
+
+    calldata_grant_role = guardian.grantRole.encode_input(DEFAULT_ADMIN_ROLE, dev_msig)
+
+    safe.submitTransaction(guardian, 0, calldata_grant_role, {"from": dev})
+
+    if sim:
+        owners = safe.getOwners()[0:2]
+        for owner in owners:
+            signer = accounts.at(owner, force=True)
+            safe.confirmTransaction(safe.transactionCount() - 1, {"from": signer})
+
+    C.print(
+        f"Calldata for granting ADMIN_ROLE to dev_msig=[blue]{calldata_grant_role}[/blue]. Target:[blue]{guardian.address}[/blue]\n"
+    )
+
+
+def main(safe_ens, sim=False):
+    # NOTE: before heading towards configuration, ensure current techops signers are in gas st for `MAINNET`
+    if chain.id == 1:
+        # watchlist in gas st
+        gas_st_watchlist = Contract(r.badger_wallets.gas_station).getWatchList()
+        for signer in techops_signers:
+            assert signer is gas_st_watchlist, f"Signer {signer} not in watchlist!"
```

Comment:
Agree, my apologies as I know that spec might have been misleading there. There's no need to enforce this.

Reply:
good point initially i put a v tight check there, but i removed it completely in dc7554ef6fcfcd4614b4936bd7dbdba67573ebd9, since those signers which are not in watchlist and included with the deployer, plus the one which was missing `ops_executor15` is already added

Reply:
signers exceptions are `ops_executor8`, `ops_executor7` & `ops_executor2` which have deployer avail for execs

only `0xfA5bb45895Cb3C0aE5B1583Fe068f009A48F0187` is left out, but checked internally its alright